newEraCaps/flows/fulfillment/wmsFeed/detailItem.md

```diff
@@ -80,6 +95,7 @@ Note: It is recommended to use the variant barcode value for this field.
 - **Type:** Numeric
 - **Description:** Represents the subtotal of the unit prices of items, including VAT, as per the Ruby code.
 - **Conclusion:** This field containts the actual unit cost of the order item, including VAT and item discount amounts.
+- **OMS Value:** Not used, leave Empty
```

Comment:
is it possible to get a date when it was decided that this would be empty? If so we would want to add it here. If not, we can proceed.

Reply:
This I have added empty according to **NEC: WMS Brokered Feed Mapping** file sent by Janvi Ma'am, which has QnA with Customer, but date is not present there.